src/MoBi.Presentation/Presenter/Main/MenuAndToolBarPresenter.cs

```diff
@@ -80,7 +80,7 @@ protected override void AddRibbonPages()
          _view.AddApplicationMenu(_buttonGroupRepository.Find(ButtonGroupIds.File));
          _view.AddPageGroupToPage(_buttonGroupRepository.Find(ButtonGroupIds.BuildingBlocks), AppConstants.RibbonPages.Modeling);
          _view.AddPageGroupToPage(_buttonGroupRepository.Find(ButtonGroupIds.Workflows), AppConstants.RibbonPages.Modeling);
-         _view.AddPageGroupToPage(_buttonGroupRepository.Find(ButtonGroupIds.Project), AppConstants.RibbonPages.Modeling);
+         _view.AddPageGroupToPage(_buttonGroupRepository.Find(ButtonGroupIds.ProjectSimulationSettings), AppConstants.RibbonPages.Modeling);
```

Comment:
Ok in Modeling. Do we have one for project

Reply:
No, we don't, but that's the full bar for 'Modeling'. Maybe this could be named 'Project'.

![image](https://user-images.githubusercontent.com/261477/230052355-a4083342-b827-4ccf-b2dd-8917bc738848.png)

Reply:
Then rename our new grouping 'Simulation Settings'
The 'Create' group will be reduced to Module, Individual, Expression, Simulation. All these are added to the 'Project' so I think it makes some sense.

Reply:
I would not rename modeling to project. Let's simply use simulation settings for now